Userland/Libraries/LibWeb/HTML/Worker.cpp

```diff
@@ -5,20 +5,16 @@
  */
 
 #include <AK/String.h>
-<<<<<<< HEAD
```

Comment:
Messed up rebase here?

Reply:
Yea, I screwed up the rebase a bit on the previous commit and didn't realize until it was too late. =/ At this point am I better off just squashing everything together into a single commit?

Reply:
Depends on how good your git-fu is! Easiest way it probably squashing it down into one commit, running `git reset HEAD^` to undo that commit but keep its changes, and start committing again from there - for example the test page can be its own commit.

Reply:
Alright, I think I did the thing!

Brought it down to 2 commits and incorporated all the fixes mentioned. I was also able to completely remove the changes to `WorkerGlobalScope` since I'm simply using `GlobalObject` directly at the moment.

Reply:
Had to push again because of an unused captured variable for a lambda that was only failing the build on the azure pipeline.